app/assets/stylesheets/application.scss

```diff
@@ -165,3 +220,31 @@
 .timeline-body > p + p {
   margin-top: 5px;
 }
+
+.apple_pagination {
```

Comment:
Seems like the indentation here is being used to show a relationship between this styling? I like the idea of using whitespace to visually communicate relationships, but I'm wary of using indentation since it can mean things in other languages (i.e. Python). Maybe keep the conventional indentation, but remove the newline whitespace between the selectors to communicate their association?

````
.apple_pagination {
    /* code */
}
.apple_pagination a, apple_pagination span {
   /* code */
}
.apple_pagination .disabled {
    /* code */
}